swift/ios-shoppe-demo/Views/CartTableViewCell.swift

```diff
@@ -11,18 +11,32 @@ import UIKit
 
 class CartTableViewCell: UITableViewCell {
     @IBOutlet weak var orderTotalLabel: UILabel!
+    @IBOutlet weak var checkoutButton: UIButton!
 
-    var tableViewController: CartTableViewController?
+    weak var cartTableViewController: CartTableViewController?
+    weak var checkoutTableViewController: CheckoutTableViewController?
+
+    func setup() {
+        setupCartOrder()
+
+        checkoutButton.curveViewCornersWithShadow()
```

Comment:
Are we doubly `checkoutButton.curveViewCornersWithShadow()` here and in `setupCartOrder`?

Reply:
i'm combining setup -> setupCartOrder